include/rest/api_client.hpp

```diff
@@ -144,19 +243,11 @@ namespace disccord
                     }
 
                     template <typename TModel>
-                    pplx::task<void> request_multi_json(route_info& route, std::vector<TModel> body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
+                    pplx::task<void> request_multi_json(route_info& route, TModel body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
```

Comment:
Models should never encode to arrays though? I'll look into the requests which require an array.